bindings/python/examples/iio_stream.py

```diff
@@ -0,0 +1,76 @@
+"""
+AD9361 IIO streaming example - Python bindings
+"""
+import iio
+import time
+import numpy as np
+import matplotlib.pyplot as plt
+
+USB_CONTEXT = "ip:192.168.2.1"
```

Comment:
The contexts suggests an Ethernet context not an USB one.

Reply:
Solved.